data/json/monsters/zed_misc.json

```
@@ -527,6 +527,130 @@
       "FILTHY"
     ]
   },
+  {
+    "id": "mon_zombie_nomex_1",
+    "type": "MONSTER",
+    "name": { "str": "nomex zombie" },
+    "description": "This was at some point a zombie wearing some kind of fire-resistant protective gear.  That same outfit is now shredded and has become part of its outer tissue, making it more resilient against physical attacks and heat-induced damage.  It also seems to be rather muscle packed.  At least its overgrown and hardened faceplate still holds up, preventing it from biting you and also rendering its eyes impaired.  However, it looks like it can still hear and smell you just fine.",
+    "default_faction": "zombie",
+    "bodytype": "human",
+    "species": [ "ZOMBIE", "HUMAN" ],
+    "diff": 2,
+    "volume": "72500 ml",
+    "weight": "85500 g",
+    "hp": 120,
+    "speed": 70,
+    "material": [ "flesh" ],
+    "symbol": "Z",
+    "color": "yellow",
+    "aggression": 100,
+    "morale": 100,
+    "melee_skill": 6,
+    "melee_dice": 3,
+    "melee_dice_sides": 3,
+    "melee_damage": [ { "damage_type": "cut", "amount": 0 } ],
+    "armor_bash": 10,
+    "armor_cut": 10,
+    "armor_bullet": 8,
+    "armor_acid": 4,
+    "armor_fire": 15,
+    "armor_elec": 5,
+    "weakpoint_sets": [ "wps_humanoid_body", "wps_humanoid_turnout_gear", "wps_humanoid_full_helmet_armored" ],
+    "families": [ "prof_intro_biology", "prof_wp_zombie", "prof_wp_syn_armored", "prof_wp_nat_armored" ],
+    "bleed_rate": 50,
+    "vision_day": 10,
+    "vision_night": 5,
+    "harvest": "zombie_nomex",
+    "special_attacks": [
+      {
+        "id": "slam",
+        "cooldown": 15,
+        "move_cost": 240,
+        "range": 1,
+        "damage_max_instance": [ { "damage_type": "bash", "amount": 12 } ],
+        "throw_strength": 10
+      },
+      {
+        "id": "headbash",
+        "cooldown": 20,
+        "move_cost": 300,
+        "damage_max_instance": [ { "damage_type": "bash", "amount": 16 } ]
+      }
+    ],
+    "upgrades": { "half_life": 42, "into": "mon_zombie_nomex_2" },
+    "death_drops": "mon_zombie_fireman_death_drops",
+    "burn_into": "mon_zombie_scorched",
+    "flags": [
+      "SEES",
+      "HEARS",
+      "STUMBLES",
+      "WARM",
+      "BASHES",
+      "GROUP_BASH",
+      "POISON",
+      "ACIDPROOF",
+      "NO_BREATHE",
+      "REVIVES",
+      "PUSH_MON",
+      "NO_FUNG_DMG",
+      "FILTHY"
+    ]
+  },
+  {
+    "id": "mon_zombie_nomex_2",
+    "type": "MONSTER",
+    "name": { "str": "nomex hulk" },
+    "description": "This was at some point a zombie wearing some kind of fire-resistant protective gear.  It has grown to an enormous size with bulging muscles and features a hide in the likeness of its former uniform.  The faceplate that once shielded its owner's face against fire has now hardened to more closely resemble a slightly curved, bone-crushing hammerhead.  As a result its ability to see is heavily impaired, but its hearing and sense of smell appear to be working just fine.",
```

```suggestion
    "description": "This was at some point a zombie wearing fire-resistant protective gear.  It has grown to an enormous size with bulging muscles and a hide in the likeness of a firefighter uniform.  The faceplate that once shielded its face has now hardened to into slightly curved, bone-crushing hammerhead.  As a result its ability to see is heavily impaired, but its hearing and sense of smell appear to be working just fine.",
```
I tried to streamline the description a bit